app/src/main/java/com/github/scribeWizTeam/scribewiz/models/UserModel.kt

```diff
@@ -4,90 +4,86 @@ import android.content.Context
 import com.google.firebase.firestore.ktx.firestore
 import com.google.firebase.firestore.ktx.toObject
 import com.google.firebase.ktx.Firebase
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.tasks.await
 
```

Comment:
Good thing to use nullable since those users won't necessarily have friends/musicNotes